package.json

```diff
@@ -18,8 +19,8 @@
     "postinstall": "yodoctor",
     "postupdate": "yodoctor",
     "pretest": "xo",
-    "test": "nyc mocha --timeout=30000",
-    "coverage": "nyc report --reporter=text-lcov | coveralls"
+    "test": "c8 mocha --timeout=30000",
+    "coverage": "c8 report --reporter=text-lcov | coveralls"
```

Comment:
* `nyc` doesn't provide coverage for ESM projects (https://github.com/istanbuljs/nyc/issues/1287)
* `esm-loader-hook` is promising but experimental (https://github.com/istanbuljs/esm-loader-hook)
* c8 is a fully-compatible replacement that worked immediately